src/Spryker/Zed/SynchronizationBehavior/Persistence/Propel/Behavior/SynchronizationBehavior.php

```diff
@@ -192,7 +193,7 @@ public function modifyTable()
                 ]);
                 $uniqueIndex = new Unique();
                 $uniqueIndex->setName($table->getName() . '-unique-mapping-resource-key');
-                $uniqueIndex->addColumn($table->getColumn('mapping_resource_key'));
+                $uniqueIndex->addColumn(new Column('mapping_resource_key'));
```

Comment:
What will be the output difference? 
In which cases it's a BC-break?

Reply:
There is no BC break as Pqsql doesn't add the size attribute anyways. This is only for MySql.

Instead of using the by the behavior added column which contains the size information we use a plain and new column.

The difference is:

Current

```
<unique name="table-name-unique-key">
    <unique-column name="key" size="255"/>
</unique>
```

New 

```
<unique name="table-name-unique-key">
    <unique-column name="key"/>
</unique>
```

Reply:
So it's a Core bug, which leaked table column size into index column definition.
Looks good then!